shell/sct_run_batch.sh

```diff
@@ -94,16 +116,15 @@ else
   list_path_subject=( "${ONLY_PROCESS_THESE_SUBJECTS[@]/#/${PATH_DATA}/}" )
 fi
 
-# Run processing with or without "GNU parallel", depending if it is installed or not
-if [ -x "$(command -v parallel)" ]; then
-  echo 'GNU parallel is installed! Processing subjects in parallel using multiple cores.' >&2
+
+# Run processing with or without "GNU parallel", depending if it is flagged and installed or not
```

Comment:
i would do the logic differently:
1. test if -s is specified: `echo Processing subjects sequentially.`
2. if not specified, check if parallel is installed
3. if yes: `echo GNU parallel is installed! Processing subjects in parallel using multiple cores.`
4. if no: `echo GNU parallel is not installed! Processing subjects sequentially.`